release-3.7.0-announcement.md

```diff
@@ -0,0 +1,31 @@
+We're excited to announce the availability of Jamulus 3.7.0!
+
+This release re-organizes Server lists to make room for more servers:
+- Default has been renamed to Any Genre 1.
+- Any Genre has been renamed to Any Genre 2.
+- Any Genre 3 has been added.
+- Choral/Barbershop has been added.
+- Classical/Folk/Choral has been renamed to Classical/Folk.
+
+Besides that, tens of usability improvements, enhancements, bugfixes and optimizations have been integrated.
+Please find all the details in the [Changelog](https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0).
+
+Downloads:
+- [Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_win.exe)
+- [Mac](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_mac.dmg)
+- [Debian/Ubuntu (amd64)](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_ubuntu_amd64.deb), alternative: [headless version](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_headless_3.7.0_ubuntu_amd64.deb)
+- [Source code](https://github.com/jamulussoftware/jamulus/archive/r3_7_0.tar.gz)
+
+This is the first release after the founder, developer and project lead of Jamulus, Volker Fischer (@corrados), [decided to leave the project](https://github.com/jamulussoftware/jamulus/discussions/928).
+We want to use this opportunity to thank Volker again. Creating Jamulus and pushing it to the point where it is today was and is a huge accomplishment! Thanks, Volker!
+
+Starting with this release, the project will be lead by the Main Developers Team.
+
+Thanks to everyone who did their part to make this release happen:
+- Code contributors: @ann0see @atsampson @dakhubgit @dcorson-ticino-com @doloopuntil @dzpex @genesisproject2020 @hoffie @ignotus666 @jerogee @jujudusud @melcon @mulyaj @nefarius2001 @npostavs @passing @pljones @ranfdev @SeeLook @Snayler @softins
+- Translators: @dzpex @genesisproject2020 @ignotus666 @jerogee @jose1711 @jujudusud @melcon @rolamos @SeeLook @Snayler @trebmuh
```

Comment:
Actually there were/are  more gegeweb who did parts of the french documentation for example.

Reply:
Then we should definitely add them. I generated the list from the list of app translation PRs. Do you have a quick way to get the additional people who did other translations such as docs?

Reply:
Maybe the contributor list on the doc repo.

Reply:
Ok, will check later. Maybe I can come up with some `git log`-based automation. Should be possible.

Reply:
There is also the contributors list in `util.cpp`, which we have tried to keep up to date and includes previous contributors.

Reply:
Turns out...
- It's a bit more than a one-liner. I'll add a PR for the `tools/` directory shortly.
- As my hand-generated list was based on PRs, I missed some people, even corrados!

I think the list in the announcement should be about the people who contributed to that release. This will be a subset of the `util.cpp` or Github contributor lists.

@ann0see @softins Can you re-check the list again now?